### PR TITLE
Allow specifying type imports for `diesel print-schema`

### PR DESCRIPTION
**Pull request by sgrif**
This adds a new CLI flag and corresponding config option for `diesel
print-schema`. This new flag will cause `use` lines to be added to the
top of `table!` declarations. As with hand-written `table!`s, if one
wishes to have `use diesel::sql_types::*` included as well, that will
need to be manually specified.

For simplicity, I have kept this limited to something that applies to
all tables, not any one specific table.

### CODE REVIEW

**Comment by sgrif** [comment]
> I assume it's not possible to ever get import-types to be an empty string? Do you want to add some simple validation that the given string parses as Rust path?

I'm personally fine with just generating invalid code if it's not a valid Rust path, mostly because I don't think we will ever generate a better error message. We'd probably just use `syn` for this, and it'll just give us "could not parse all tokens" for an error message which isn't exactly helpful.

I'm not sure I understand the empty string question

**Comment by sgrif** [comment]
ping @diesel-rs/contributors This has sat around for a bit

**Comment by sgrif** [comment]
ping @diesel-rs/reviewers  This has sat around for a bit

File: diesel_cli/src/print_schema.rs [hunk 1]
```diff
@@ -82,13 +83,14 @@ impl<'a> Display for ModuleDefinition<'a> {
     }
 }
 
-struct TableDefinitions {
+struct TableDefinitions<'a> {
     tables: Vec<TableData>,
     fk_constraints: Vec<ForeignKeyConstraint>,
     include_docs: bool,
+    import_types: Option<&'a [String]>,
```
[the remark this answers is not in the document]
Comment (sgrif) [reply diesel_cli/src/print_schema.rs]: I actually think there'd be more noise if I took a vec

[the remark this answers is not in the document]
Comment (sgrif) [reply diesel_cli/src/print_schema.rs]: I guess maybe it'd just be a `.to_owned`, but we are talking 3 lines vs 2...

File: diesel_cli/tests/print_schema/print_schema_custom_types/print_schema_simple_without_docs/diesel.toml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[print_schema]
```
Comment (weiznich) [inline diesel_cli/tests/print_schema/print_schema_custom_types/print_schema_simple_without_docs/diesel.toml]: Those files seems to be in the wrong place?
`diesel_cli/tests/print_schema/print_schema_custom_types/print_schema_simple_without_docs/diesel.toml`
seems to be wrong. 
Maybe they belong to 
`diesel_cli/tests/print_schema/print_schema_simple_without_docs/diesel.toml`?

**Pull request merged**